libs/libjson/_internal/Source/JSONNode.h

```diff
@@ -79,11 +79,12 @@
     #endif
 
     #define DECLARE_FOR_ALL_TYPES(foo)\
-	   foo(char) json_nothrow;	 foo(unsigned char) json_nothrow;\
-	   foo(short) json_nothrow;	 foo(unsigned short) json_nothrow;\
-	   foo(int) json_nothrow;	 foo(unsigned int) json_nothrow;\
-	   foo(long) json_nothrow;	 foo(unsigned long) json_nothrow;\
-	   foo(float) json_nothrow;	 foo(double) json_nothrow;\
+       foo(char) json_nothrow;      foo(unsigned char) json_nothrow;\
```

Comment:
Found the problem: Github seems to visualize 8 spaces for a tab. I do 4. 

![screen shot 2016-01-28 at 16 58 48](https://cloud.githubusercontent.com/assets/282446/12649830/7f8b7c90-c5e0-11e5-84e3-e4a44f631c53.png)

I propose we switch to clang-formatter?